ms-common-auth/src/main/java/net/trajano/ms/auth/spi/ClientValidator.java

```diff
@@ -3,7 +3,10 @@
 import java.net.URI;
 import java.text.ParseException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.text.ParseException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)